src/logger/src/logger.rs

```diff
@@ -334,10 +334,10 @@ impl Logger {
     }
 
     /// if the max level hasn't been configured yet, set it to default
-    fn try_init_level(&self) {
+    fn try_init_max_level(&self) {
```

Comment:
nit: `try_` functions typically return a Result. You can look at examples here: https://doc.rust-lang.org/nightly/core/index.html?search=try_